src/index.js

```diff
@@ -0,0 +1,83 @@
+import { getLikes, postLikes } from './modules/interaction.js';
+import {
+  input, submitComment, textarea, close, submit, openComments,
+} from './modules/comments.js';
+import './style.css';
+
+const apiBaseUrl = 'https://api.tvmaze.com';
+
+function loadData() {
```

Comment:
- [ ] Good job working on this but in order to stick to using ES6, please kindly use an arrow function here.